src/preferences/dialog/dlgprefwaveform.cpp

```diff
@@ -284,9 +284,9 @@ void DlgPrefWaveform::slotUpdate() {
     updateEnableUntilMark();
     updateWaveformGeneralOptionsEnabled();
 
-    bool disableEqGain = m_pConfig->getValue(
-            ConfigKey("[Waveform]", "disable_eq_gain"), false);
-    disableEqGainCheckBox->setChecked(disableEqGain);
+    bool visualizeEqGain = m_pConfig->getValue(
+            ConfigKey("[Waveform]", "visualize_eq_gain"), false);
```

Comment:
lets default this to true since that was the default previously.
```suggestion
    bool visualizeEqGain = m_pConfig->getValue(
            ConfigKey("[Waveform]", "visualize_eq_gain"), true);
```